### ISSUE TITLE
Win10: PopUnder bug

### ISSUE THREAD

**Issue by drdksebc** [keybase/client#5909]
Keybase GUI Version: 1.0.18-20170208085427+dc66461

Hi There,

Whenever I click on the icon in the Windows Tray, then the PopUp menu stays behind the tray and all the other icons, so I have to move the cursor out and let the tray dispensary or click on the menu for it to get to the front....

![keybase popupunder](https://cloud.githubusercontent.com/assets/25318765/23068199/fdda0086-f522-11e6-9723-211d02a7bdb9.PNG)

**Comment by Juvls** [comment]
Yes I have the same issue! Annoying but if you wait a bit, the popup will disapear.

**Comment by thomas-klein** [comment]
I have a variant of this problem because of my dual screen setup I guess. 
The menu pop between my 2 screens and under the taskbar making it impossible to see the whole interface.
![dual_screen_menu_overlap](https://cloud.githubusercontent.com/assets/3647800/23546811/96d42396-0001-11e7-9457-0d4317c5f2f9.png)
Context : my taskbar is vertical on the left of my right screen.

my log id: c15f625c59418d60bc3b7c1c
